modbusApp/src/modbusInterpose.c

```diff
@@ -15,6 +15,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <stdbool.h>
 
```

Comment:
If we want to submit this upstream, then I think we need to avoid C99 i.e. no stdbool just int

Reply:
ok no probs, i can just use int

Reply:
its' a bit like in EPICS C++ world, currently if you want to submit upstream you need to keep visual studio 2010 compatible and no C++11